Docs/reference/content/reference/driver/ssl.md

```diff
@@ -72,9 +72,12 @@ Prior to v2.7.0, the driver enabled certificate revocation checking by default,
 |         | .NET Core 2.1         | Yes    | Yes    | Yes | No                |
 
 
-#### Notes:
- - SNI (Server Name Indication) required for Atlas free tier.
- - If a server's certificate includes Certificate Revocation List (CRL) Distribution Points but does not include an Online Certificate Status Protocol (OCSP) extension, .NET Core on OSX will fail to connect due to a limitation of the Apple Security Framework (see https://github.com/dotnet/corefx/issues/29064). Prior to version 2.0, .NET Core on OSX used OpenSSL, which does support CRLs without OCSP.
+#### Notes
+ - SNI (Server Name Indication) is required for Atlas free tier.
+ - .NET Core on OSX will fail to connect if **both** of the following conditions are met: (1) [certificate revocation checking]({{<relref "reference\driver\ssl.md#certificate-revocation-checking" >}}) is enabled and (2) a server's certificate includes Certificate Revocation List (CRL) Distribution Points but does not include an Online Certificate Status Protocol (OCSP) extension.
+ 
+  - This is due to a limitation of the Apple Security Framework (see https://github.com/dotnet/corefx/issues/29064). Prior to version 2.0, .NET Core on OSX used OpenSSL, which does support CRLs without OCSP. 
+  - Connecting to Atlas on OSX with certificate revocation checking enabled will succeed since Atlas certificates include CRL Distribution Points as well as an OCSP extension.
```

Comment:
Thought it might be nice to clarify that the limitation isn't in play when working with Atlas.